### DISCUSSION TITLE
Build Failing on Linux armv7

### DISCUSSION THREAD

**Discussion by vednig** [zed-industries/zed#22485] [Feature Requests]
error: failed to run custom build command for `cranelift-codegen v0.111.2`

Caused by:
  process didn't exit successfully: `/home/master/Downloads/zed/target/debug/build/cranelift-codegen-03fbe87bf68a4c72/build-script-build` (exit status: 101)
  --- stderr
  thread 'main' panicked at /home/master/.cargo/registry/src/index.crates.io-1cd66030c949c28d/cranelift-codegen-0.111.2/build.rs:50:53:
  error when identifying target: "no supported isa found for arch `armv7`"
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
